js/src/utils/getConversionCampaignStatusNotice.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Internal dependencies
+ */
+import { CAMPAIGN_ADS_MIGRATION, MONTH_IN_SECONDS } from '.~/constants';
+
+/**
+ * Returns the conversion campagin status
+ *
+ * @param {Object} adsCampaignConvertStatus wp option with the information related to the conversion status
+ * @param {string} adsCampaignConvertStatus.status Conversion status: unconverted|converted|not-applicable|unknown
+ * @param {number} adsCampaignConvertStatus.updated Unix time in seconds of the last time that the status was checked. Once is converted the timestamp is not updated anymore.
+ * @param {number} timestamp Timestamp that is used as a reference
+ * @return {string} return the content to show in the banner
+ */
+const getConversionCampaignStatusNotice = (
+	adsCampaignConvertStatus,
+	timestamp = new Date().getTime()
+) => {
+	const isConverted = adsCampaignConvertStatus?.status === 'converted';
+	const conversionTimestamp = adsCampaignConvertStatus?.updated;
+
+	const isInMigrationPeriod =
+		conversionTimestamp &&
+		timestamp - conversionTimestamp * 1000 <
+			CAMPAIGN_ADS_MIGRATION.MAXIMUM_MONTHS_TO_DISPLAY_NOTICE *
+				MONTH_IN_SECONDS *
+				1000;
+
+	if ( ! isConverted ) return 'BEFORE_CONVERSION';
+	else if ( isConverted && isInMigrationPeriod ) {
```

Comment:
💅 

I think we could just write `else if ( isInMigrationPeriod ) {` as `isConverted` will be true here. But I'm also fine to keep it as is since it's more explicit.

Reply:
Yeah, it is true, it is redundant the isConverted. Updated [70204c2](https://github.com/woocommerce/google-listings-and-ads/pull/1472/commits/70204c2c8573482e63328e832d8713aa1dfcff00)